service/src/test/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/OrganizationMapperTest.java

```diff
@@ -52,11 +61,54 @@
     @Mock
     private II ii;
 
+    @Mock
+    private POCDMT000002UK01InformationRecipient informationRecipient;
+
+    @Mock
+    private POCDMT000002UK01IntendedRecipient intendedRecipient;
+
+    @Mock
+    private Node node;
+
+    @Mock
+    private ON on;
+
     @Mock
     private ResourceUtil resourceUtil;
 
+    @Test
+    public void shouldMapOrganizationInformationRecipient() {
+
+        POCDMT000002UK01Organization itkOrganization = mockItkOrganization();
+
+        when(informationRecipient.getIntendedRecipient()).thenReturn(intendedRecipient);
+        when(intendedRecipient.getReceivedOrganization()).thenReturn(itkOrganization);
+        when(intendedRecipient.isSetReceivedOrganization()).thenReturn(true);
+        lenient().when(informationRecipient.getTypeCode()).thenReturn(forString(PRCP_TYPE_CODE));
```

Comment:
Why lenient? In this case it seems unnecessary. I'd avoid lenient mocking in general - unless we have a good reason.